rwengine/src/engine/GameWorld.hpp

```diff
@@ -401,7 +401,8 @@ class GameWorld {
         typename std::enable_if<std::is_integral<T1>::value>::type* = nullptr,
         typename std::enable_if<std::is_integral<T2>::value>::type* = nullptr>
     T1 getRandomNumber(T1 min, T2 max) {
-        std::uniform_int_distribution<T1> dist(min, max);
```

Comment:
I think making the `std::uniform_int_distribution` static makes the code unclear.
It is also a premature optimization.
Do we know whether the construction cost of a `uniform_int_distribution` is high?
Because in the next line, it is copy constructed anyways.

Reply:
reverted